examples/kmm-sample/iosApp/iosApp/ContentView.swift

```diff
@@ -35,6 +60,12 @@ struct ContentView: View {
                 Text(sum)
             }
             Text("History count: " + String(count))
+        }.onAppear() {
+            print("onAppear")
```

Comment:
Up to you if you want to leave print statements in the merged code, but I'd probably take them out